fmriprep/workflows/bold/registration.py

```diff
@@ -18,6 +18,7 @@
 from nipype.pipeline import engine as pe
 from nipype.interfaces import utility as niu, fsl, c3, freesurfer as fs
 from niworkflows.interfaces.registration import FLIRTRPT, BBRegisterRPT, MRICoregRPT
```

Comment:
For `BBRegisterRPT` and `MRICoregRPT`, we should import along with `PatchedConcatenateLTA`, on line 30, below:

```Python
from ...interfaces.freesurfer import (
    PatchedMRICoregRPT as MRICoregRPT,
    PatchedBBRegisterRPT as BBRegisterRPT,
    PatchedConcatenateLTA as ConcatenateLTA)
```

Reply:
ok, 
is it not more logical then to put TuncateLTRA in interface/freesufer
but PatchedBBRegisterRTP and PatchedMRICoregRPT in the interface/registration ( juste after the definition of BBRegisterRPT nad MRICoregRPT

Reply:
oups I forget that BBRegisterRTP is in niworkflow,
so to stay in fmriprep, I add then as you said
_